src/main/java/via/dk/cueandbrew/model/Reservation.java

```diff
@@ -0,0 +1,82 @@
+package via.dk.cueandbrew.model;
+
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Reservation {
+    private String notes;
+    private String clientFirstName;
+    private String clientLastName;
+    private String clientPhoneNumber;
+    private Timestamp creationDatetime;
+    private List<Booking> booking;
+    private List<Order> order;
```

Comment:
I think there should be 1 instance of Order not an Array, because 1 order contains more drinks and it is not relevant to have more drink orders when we reserve a table. Plus there is no possibility in the design to make so